app/components/listing/ResourceInfos.jsx

```diff
@@ -89,7 +89,7 @@ AddressInfo.propTypes = {
 
 const PhoneNumber = ({ phones }) => (
   <span className="phone">
-    { phones.map(phone => <p key={phone.id}>{`${phone.number} ${phone.service_type}`}</p>) }
+    { phones.map(phone => <p key={phone.id}><a href={`tel:${phone.number}`}>{`${phone.number} ${phone.service_type}`}</a></p>) }
```

Comment:
Minor suggestion: I think I'd probably make just the phone number itself clickable, not the service type as well, since I feel like that's usually how websites present clickable phone numbers.

Reply:
huh, never thought of that! No problem though let me fix that :) Can I still use the same branch and same pull request?